exercise1/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using Newtonsoft.Json;
+using RestSharp;
+using Retrofit.Net;
+
+namespace exercise1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var dataCenter = "us17";
+            var apiKey = "ed37a130dbadee9671f725ebdf03583a-us17";
+            var listId = "ea602c2f5c";
+            var campaignId = "39494cd8ea";
+            var url = $"https://{dataCenter}.api.mailchimp.com/3.0";
+            // init service
+            var service = GetService(url, apiKey);
+            // create a new list
+            var list = GetListObject();
+            Console.WriteLine(service.CreateList(list).ResponseStatus);
+            // add an email to list
+            var emailSubcriber = GetEmailObject("quoctran2124@gmail.com");
+            Console.WriteLine(service.AddEmail(listId,emailSubcriber));
+            // create a new campaign
+            var campaign = GetCampaignObject(listId);
+            Console.WriteLine(service.CreateCampaigns(campaign).ResponseStatus);
+            // set content for a campaign
+            var content = GetContentObject("Hello there!");
+            Console.WriteLine(service.SetContent(campaignId,content));
+            // send campaign
+            Console.WriteLine(service.SendCampaign(campaignId));
+            Console.ReadKey();
+        }
+        
+        private static IHelper GetService(string url, string apiKey)
+        {
+            var restClient = new RestClient(url);
+            restClient.Authenticator = new HttpBasicAuthenticator("username", apiKey);
+            var adapter = new RestAdapter(restClient);
+            return adapter.Create<IHelper>();
+        }
+
+        private static Object GetListObject()
```

Comment:
Move this method (and all methods below) to a new class.

Reply:
done